src/main/java/com/genfood/foodgenback/endpoint/rest/controller/PingController.java

```diff
@@ -0,0 +1,15 @@
+package com.genfood.foodgenback.endpoint.rest.controller;
+
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@AllArgsConstructor
+public class PingController {
+
+  @GetMapping("/ping")
+  public String HealthController() {
```

Comment:
it is a function not a class so it should begin with a lowercase.
HealthController -> healthController